subsys/testsuite/include/ztest.ld

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+SECTIONS
+{
+	.data.ztest_suite_node_area : ALIGN(4)
+  	{
```

Comment:
Mixing SPACEs and TABs here. Should stick with TABs.

Reply:
Thanks, snuck past me :)